tasks/docker_network.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Create a network
+  docker_network:
+    name: "{{ item.name }}"
+    connected: "{{ item.containers }}"
```

Comment:
Is there a way to not specify this here, but in each container stack?  This diverges from the pattern of everything for each stack being in one place. 

yes, it looks like you could do that instead: 
https://docs.ansible.com/ansible/latest/modules/docker_container_module.html

So I would recommend to not specify networks here.  

Also, additional docs will be necessary to show the new network functionality from creating the network, as well as connecting individual containers to a network.